datagrid/datagrid72-image-stream.json

```diff
@@ -0,0 +1,125 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "datagrid72-image-streams",
+        "annotations": {
+            "description": "ImageStream definitions for Red Hat JBoss Data Grid 7.2.",
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-datagrid72-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "Red Hat JBoss Data Grid 7.2",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "1.4.10"
+                } 
+            },
+            "labels": {
+                "xpaas": "1.4.10"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "1.0",
```

Comment:
I believe we need only `1.0` version tag. Looking at https://issues.jboss.org/browse/CLOUD-2306, you may find (the last comment) that the latest image version is `jboss-datagrid-7/datagrid72-openshift:1.0` (omit the registry name, that will change once RCM push it). So please delete all other tags and leave just 1.0.